File: pset6/mario/mario.py
from cs50 import get_int

# Condicao para garantir que o usuario digite entre 1 e 8
while True:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break

# Loop para adicionar a altura
for i in range(n):
    # Loop para adicionar a # e tbm o espaço
    for j in range(n):
        if i + j >= n - 1:
            print("#", end="")
        else:
            print(" ", end="")
    print()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>


int main(void)
{
    int height, space, i, a; //declaração do tipo das variaveis
    do
    {
        height = get_int("altura: ");
    }
    while (height < 1 || height > 8); //loop para que a resposta esteja dentro dos criterios

    for (i = 0; i < height; i++)
    {
        for (space = 0; space < height; space++) //loop que adiciona os espaços entre as #
        {
            if (i + space < height - 1) //condição que verifica se ainda terá espaço em branco
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}